src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -273,24 +269,19 @@ private void testAddAction() throws Exception {
                 newSession.getInstructions(), newSession.getGracePeriodMinutes());
         feedbackPage.waitForTextsForAllStatusMessagesToUserEquals(Const.StatusMessages.FEEDBACK_SESSION_EXISTS);
 
-        ______TS("success case: private session, boundary length name, only results email");
+        ______TS("success case: boundary length name, only results email");
 
         CourseAttributes otherCourse = testData.courses.get("course");
 
         feedbackPage = getFeedbackPageForInstructor(idOfInstructorWithSessions);
         feedbackPage.clickEditUncommonSettingsButtons();
-        feedbackPage.clickNeverVisibleTimeButton();
 
-        //verify that timeFrameTable, instructions and ResponseVisTable are all hidden
-        assertTrue(feedbackPage.isHidden(By.id("timeFramePanel")));
-        assertTrue(feedbackPage.isHidden(By.id("responsesVisibleFromColumn")));
-        assertTrue(feedbackPage.isHidden(By.id("instructionsRow")));
-
-        newSession.setFeedbackSessionName("private session of characters1234567 #");
+        newSession.setFeedbackSessionName("standard session of characters12345678");
```

Comment:
Remove standard. It hints that there used to be another type of session other than standard.